source/kmip.rst

```diff
@@ -26,16 +26,13 @@ KMIP enables the communication between key management systems and the database s
         - The hostname or IP address of the KMIP server.
       * - --kmipPort
         - number
-        - The port used to communicate with the KMIP server. 
+        - The port used to communicate with the KMIP server. When undefined, the default port 5696 will be used.
       * - --kmipServerCAFile
         - string
         - The path to the CA certificate file. CA file is used to validate secure client connection to the KMIP server.
       * - --kmipClientCertificateFile
         - string
-        - The path to the client certificate file. The database server uses the client certificate file to authenticate the KMIP server.
-      * - --kmipClientKeyFile
-        - string
-        - The path to the KMIP client private key.
+        - The path to the client certificate file and the private key. The database server uses this certificate chain to authenticate the KMIP server.
```

Comment:
I would say "The path to the PEM file containing both client certificate and private key."